hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/BackendStore.java

```diff
@@ -101,11 +101,9 @@ default Id nextId(HugeType type) {
             }
         }
 
-        E.checkState(counter != 0L, "Please check whether '%s' is OK",
-                     this.provider().type());
+        E.checkState(counter != 0L, "Please check whether '%s' is OK", this.provider().type());
 
-        E.checkState(counter == expect, "'%s' is busy please try again",
-                     this.provider().type());
+        E.checkState(counter == expect, "'%s' is busy please try again", this.provider().type());
```

Comment:
Can we limit line length to 80 chars as much as possible? We want to define 100 as the maximum line length, rather than guide length.

Reply:
I think we need to set forceStrictCondition=false for Indentation check

such as 
![2](https://user-images.githubusercontent.com/13429553/165665475-b20dfca6-85b7-4008-8663-bacc05e3c1b8.JPG)

Reply:
We already have a lot of code in this style, and we also recommend keeping alignment when indented. So it's a good choice to set `forceStrictCondition=false`.